shell/platform/android/io/flutter/embedding/android/AndroidTouchProcessor.java

```diff
@@ -188,15 +194,15 @@ private void addPointerForIndex(
 
     long timeStamp = event.getEventTime() * 1000; // Convert from milliseconds to microseconds.
 
-    packet.putLong(0); // motionEventId
+    packet.putLong(motionEventId); // motionEventId
     packet.putLong(timeStamp); // time_stamp
     packet.putLong(pointerChange); // change
     packet.putLong(pointerKind); // kind
     packet.putLong(signalKind); // signal_kind
     packet.putLong(event.getPointerId(pointerIndex)); // device
     packet.putLong(0); // pointer_identifier, will be generated in pointer_data_packet_converter.cc.
-    packet.putDouble(event.getX(pointerIndex)); // physical_x
-    packet.putDouble(event.getY(pointerIndex)); // physical_y
+    packet.putDouble(event.getRawX(pointerIndex)); // physical_x
+    packet.putDouble(event.getRawY(pointerIndex)); // physical_y
```

Comment:
As discussed, we would need a shim for `getRawX` and `getRawY`.

Reply:
Yup, looking into it now.